reader/reader/core/reader_service.py

```diff
@@ -1,3 +1,10 @@
+from typing import Any, Dict, List, Optional
+
+from flask import jsonify
```

Comment:
Plase do not import flask outside of the frontend (this loads more in the background and creates a huge dependency)